toc.md

```diff
@@ -824,7 +826,17 @@ We also decided to compare the results of JRuby vs MRI. Because jRuby can run in
 
 ## Conclusion
 
-It was a lot of fun to work on this project. We learned a lot about threaded code, memory usage by different types of OS entities, Redis data structures and other Ruby interpreters. We really hope our findings will help other developers in their journey through the pathways of threaded code.
+It was a lot of fun to work on this project! Our goal was to share potentially useful knowledge with the community and to introduce background job processors to people unfamiliar with them. After all it's usually a very interesting enterprise to understand how something works under the hood!
```

Comment:
don't be soft or shy on how useful this project is! :) Let's be confident and market how awesome it is. Aka, I'd remove words like "potentially useful" or "people unfamiliar with them" and say that whoever reads this will benefit from what we showcase. (unless it's Mike Perham maybe :D)